README.md

```diff
@@ -107,9 +110,21 @@ Onepay.checkout(options);
 ````
 
 ## Integración QR Directo
-### Crear requerimiento
+### 1. Crear transacción
 
-Lo primero que debes crear es el objeto de requerimiento para el SDK el cual se arma de la siguiente forma:
+Para comenzar a utilizar el sdk como QR Directo es necesario que crees una transacción desde tu backend y obtengas los
+datos de esta ya que serán utilizados como parámetros para que el sdk pueda comenzar a trabajar con el QR.
+
+Los datos que necesitarás son `occ`, `ott`, `externalUniqueNumber` y `qrCodeAsBase64`. Todos ellos estarán en la
```

Comment:
Please rewrite this paragraph so that the explanation follows the natural order: getting the channel on the frontend, then creating the transaction using that channel and finally getting the occ, ott, eun, qr to go to step 2. The current explanation jumps around in a non-natural order, making it harder to understand than what it needs to be.